### PR TITLE
Add information about Programs filter to docs

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3603](https://openedx.atlassian.net/browse/DOC-3603)

This PR adds information about the new Programs filter on the Courses page in Insights to the Using edX Insights guide.

### Date Needed (optional)

8 May 2017 (approx.)

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @thallada  
- [ ] Doc team review (copy edit): @edx/doc
- [ ] Product review: @katymyw 

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by dsjen** [comment]
CC: @edx/educator-dahlia

File: docs/en_us/dashboard/source/Courses_Page.rst [hunk 1]
```diff
@@ -113,7 +124,13 @@ and pacing type, two different counts for every enrollment mode, and other
 information.
 
 To download the Course Summary report in a comma-separated value (CSV) file,
-select **Download CSV**. The CSV file contains the following columns.
+select **Download CSV**.
+
+.. note::
+  The Course Summary report contains information for all of your courses, even
+  if you select filters when you view the Course List table.
```
Comment (thallada) [inline docs/en_us/dashboard/source/Courses_Page.rst:131]: :+1: thanks for this, I think this can be confusing to people

**Review by thallada — APPROVED** [review]
Everything here looks good to me!

File: docs/en_us/dashboard/source/Courses_Page.rst [hunk 2]
```diff
@@ -134,6 +151,8 @@ select **Download CSV**. The CSV file contains the following columns.
 * enrollment_modes.verified.count
 * enrollment_modes.verified.cumulative_count
 * pacing_type
+* program_ids
```
Comment (catong) [inline docs/en_us/dashboard/source/Courses_Page.rst:154]: Are these new column names really plural? "program_ids" or "program_id"? All other columns are singular.

File: docs/en_us/dashboard/source/Courses_Page.rst [hunk 3]
```diff
@@ -103,6 +103,17 @@ also combine any of these options.
   to include in your search. The list of courses updates automatically when you
   select or clear an option.
 
+====================================
+Access Data for Courses in a Program
+====================================
+
+To access Insights data for the courses in one or more specific XSeries,
```
Comment (catong) [inline docs/en_us/dashboard/source/Courses_Page.rst]: I wouldn't make this list of program types sound so much like a definitive list. "for the courses in one or more specific programs, such as XSeries or MicroMasters programs,..."

If you mention them, note that the correct name is "Professional Certificate" program (not Professional Education).

File: docs/en_us/dashboard/source/Courses_Page.rst [hunk 4]
```diff
@@ -103,6 +103,17 @@ also combine any of these options.
...
+MicroMasters, or Professional Education programs, locate **Programs** in the
+left pane, and then select the program or programs that you want. The courses
+in the program or programs then appear in the Course List table. You can use
+the Programs filter in conjunction with other filters or searches to find the
+specific courses you're interested in.
```
Comment (catong) [inline docs/en_us/dashboard/source/Courses_Page.rst]: "you're" -> "you are"

**Review by catong — COMMENTED** [review]
Some questions

File: docs/en_us/dashboard/source/Courses_Page.rst (hunk 2 above)
Comment (srpearce) [reply docs/en_us/dashboard/source/Courses_Page.rst:154]: Yep—I copied them from the CSV.

**Pull request merged**